docs/HDRUK/3.0.0.form.json

```diff
@@ -1841,5 +1841,6 @@
                   },
                   "location": "omics.platform"
             }
-      ]
+      ],
+      "url_regex": "^\\s*((https?:\\/\\/)*([a-zA-Z0-9-]+\\.?)+[a-zA-Z]{2,}(:\\d+)?(\\/[^\\s]*)?(\n)?)+$"
```

Comment:
Tried some rather convoluted urls, and this regex appears to be resilient: https://regex101.com/r/a31iK3/2